cartoframes/data/catalog/do.py

```diff
@@ -0,0 +1,13 @@
+from cartoframes.data.catalog.category import Categories
+from cartoframes.data.catalog.country import Countries
+
+
+class DO(object):
+
+    @property
+    def countries(self):
+        return Countries.get_all()
```

Comment:
I tried to have the properties initialized in the __init__ method but then I found that mocks were not working in the tests and it tried to execute the real queries (maybe mocks don't work in __init__ time??). This was fixed with this alternative.